Objects/Objects/BuiltElements/AdvanceSteel/AdvanceSteelStraightBeam.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Speckle.Core.Kits;
+
+namespace Objects.BuiltElements.AdvanceSteel
+{
+  public class AdvanceSteelStraightBeam : AdvanceSteelBeam
+  {
+    [SchemaInfo("AdvanceSteelStraightBeam", "Creates a Advance Steel straightBeam.", "Advance Steel", "Structure")]
```

Comment:
The schema info props are used in Grasshopper to dynamically generate nodes via which the users can create "native elements" for Revit, AdvanceSteel etc.. Since this class/constructor doesn't have any props to then create a native AdvanceSteelStraightBeam we should omit it for now.
See: https://speckle.guide/user/grasshopper.html#grasshopper-bim

Reply:
ok. I will do that